### PR TITLE
Added MultiManagerProvider

### PR DESCRIPTION
**Pull request by jordan-schnur**
A SingleManagerProvider was not enough for a more complex application using more than one entity manager in the cli.

### CODE REVIEW

**Comment by derrabus** [comment]
Please add tests.

File: lib/Doctrine/ORM/Tools/Console/EntityManagerProvider/MultiManagerProvider.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+namespace Doctrine\ORM\Tools\Console\EntityManagerProvider;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\Console\EntityManagerProvider;
+
+use function array_keys;
+
+final class MultiManagerProvider implements EntityManagerProvider
+{
+    /** @var EntityManagerInterface[] */
```
Comment (derrabus) [inline lib/Doctrine/ORM/Tools/Console/EntityManagerProvider/MultiManagerProvider.php:14]: To make Psalm happy:

```suggestion
    /** @var array<string, EntityManagerInterface> */
```

Comment (derrabus) [inline lib/Doctrine/ORM/Tools/Console/EntityManagerProvider/MultiManagerProvider.php:14]: To make Psalm happy:

```suggestion
    /** @var array<string, EntityManagerInterface> */
```

File: tests/Doctrine/Tests/ORM/Tools/Console/EntityManagerProvider/MultiManagerProviderTest.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+{
+    /** @var MultiManagerProvider */
+    private $multiManagerProvider;
+
+    /** @var EntityManagerInterface[] */
+    private $entityManagers;
+
+    protected function setUp(): void
+    {
+        parent::setUp(); // TODO: Change the autogenerated stub
```
Comment (derrabus) [inline tests/Doctrine/Tests/ORM/Tools/Console/EntityManagerProvider/MultiManagerProviderTest.php]: TODO: Remove the comment.

**Review by derrabus — CHANGES_REQUESTED** [review]
Thank you for the test. However, that test does not really tell me that an instance of your provider can be used in a console application. Since that is the main purpose of such a provider, we should have a test that covers this scenario.

**Comment by jordan-schnur** [comment]
> Thank you for the test. However, that test does not really tell me that an instance of your provider can be used in a console application. Since that is the main purpose of such a provider, we should have a test that covers this scenario.

Sorry for taking awhile. The test are updated.

**Review by derrabus — APPROVED** [review]

**Review by derrabus — CHANGES_REQUESTED** [review]
Your test seems to be failing in our CI. Can you please check?

**Comment by jordan-schnur** [comment]
> Your test seems to be failing in our CI. Can you please check?

@derrabus Finally back, sorry for taking so long. Can we rerun the CI pipeline so I can get the logs back? Everything passes locally so I am confused as to where the error is occurring.

**Pull request closed without merging**